litellm/types/utils.py

```diff
@@ -457,14 +457,19 @@ def __init__(
         prompt_tokens: Optional[int] = None,
         completion_tokens: Optional[int] = None,
         total_tokens: Optional[int] = None,
+        cache_creation_input_tokens: Optional[int] = None,
+        cache_read_input_tokens: Optional[int] = None,
         **params,
     ):
         data = {
             "prompt_tokens": prompt_tokens or 0,
             "completion_tokens": completion_tokens or 0,
             "total_tokens": total_tokens or 0,
         }
```

Comment:
this usage block is what i'd expect to be updated with having: 
>  cache_creation_input_tokens: Optional[int] = None,
        cache_read_input_tokens: Optional[int] = None,

since they're litellm-specific, not openai-specific params